daily_milestone.go

```diff
@@ -199,10 +217,11 @@ func createMilestones(baseURL string, token string, projectID string, milestones
 
 func main() {
 	// Declaring variables for flags
-	var token, baseURL, namespace, project string
+	var token, baseURL, namespace, project, milestoneTime string
 	var advance int
 	// Command Line Parsing Starts
 	flag.StringVar(&token, "Token", "jGWPwqQUuf37b", "Gitlab api key/token.")
```

Comment:
For the description of the flag you only sometimes use a "." at the end. Be consistent.

Reply:
There is still one dot left ;)